src/src/com/microsoft/aad/adal/AuthenticationContext.java

```diff
@@ -1430,22 +1430,30 @@ private AuthenticationResult localFlow(CallbackHandler callbackHandle,
 
         // Trying to find refresh token first, if existed, will try to use the refresh token. 
         Logger.v(TAG, "Checking refresh tokens");
-        final boolean useMultiResourceRefreshToken = cachedItem == null || cachedItem.getIsMultiResourceRefreshToken();
-        final RefreshItem refreshItem = getRefreshToken(request, useMultiResourceRefreshToken);
+        final RefreshItem refreshItem = getRefreshToken(request);
         AuthenticationResult authResult = null;
         if (!promptUser(request.getPrompt()) && refreshItem != null
                 && !StringExtensions.IsNullOrBlank(refreshItem.mRefreshToken)) {
-            Logger.v(TAG, "Refresh token is available and it will attempt to refresh token");
+            Logger.v(TAG, "Refresh token is available and it will attempt to redeem access token.");
```

Comment:
"redeem access token" isn't accurate, we never do that in ADAL, it's the calling app that redeems it against the resource. "use it to get a new access token" is more accurate, or "use it to refresh our access token" if you want.